webpack/scenes/Subscriptions/components/SubscriptionsTable/EntitlementsInlineEditFormatter.js

```diff
@@ -34,20 +34,28 @@ export const entitlementsInlineEditFormatter =
       );
     },
     renderEdit: (value, additionalData) => {
-      const { availableQuantity, availableQuantityLoaded } = additionalData.rowData;
+      const {
+        availableQuantity, availableQuantityLoaded, available,
```

Comment:
I think we might need to rename some variables here - particularly `availableQuantity` and/or `available` because they are so similar that it's not apparent what they mean. Can you define each one on this review and we'll see if we can agree on more accurate or descriptive names?

Reply:
Looks like those values are returned from additionalData.rowData and correspond to those variable names stored within the rowData. Are you suggesting I change them there as well?

Reply:
Let's 'define' them first and then see how feasible it is to change because right now it's just confusing to look at the code - not your fault!

Reply:
Since you are deconstructing variables from an object here, there is syntax to rename them if it doesn't make sense to change the names higher up the call stack.

i.e.
```javascript
> sandwich
{ meat: 'ham' }
> const { meat: animalProduct } = sandwich
undefined
> animalProduct
'ham'
```
Not sure if it helps int his case, but it is a helpful, yet underutilized es6 feature

Reply:
Here's what I thought:

availableQuantity => remainingQuantity - This is the remaining quantity of subscriptions that the user can claim for their total quantity of subscriptions.

availableQuantityLoaded => remainingQuantityLoaded - for consistency

available => currentQuantity - This is the current quantity of subscriptions that the user has allocated.

totalAvailable - This is the total available subscriptions that the user can allocate.

Reply:
> available => currentQuantity - This is the current quantity of subscriptions that the user has allocated.

ah! The 'available' value is totally misnamed! I'm guessing it's coming out of an API somewhere - haven't dug into it though. Let's go with currentQuantity as you suggested. The others can probably stay the same unless you feel strongly about renaming those ones too.